docs/howto/import.md

```diff
@@ -125,7 +125,14 @@ Once the import is complete, the changes are merged into the destination branch.
 
 #### Notes
 
-* Import uses the `src-wins` merge strategy. Therefore - import of existing objects nad prefixes in destination will override them.
+* Import uses the `src-wins` merge strategy. Therefore - import of existing objects and prefixes in destination will override them. 
+Meaning, any files that exist in the source but not in the destination will be deleted after the import. 
```

Comment:
That's not 100% accurate as @N-o-Z mentioned. Also - we don't really explain the rational behind 1 import source to 1 destination prefix, just a way to overcome it. I'm not 100% sure this hack would work if the user would have to reimport something. @ozkatz I think we need to share our import<->prefix strategy first as we designed it while considering different import use-cases in mind.

Here's my suggestion for that section:
```suggestion
### Import Strategy & Scale

The lakeFS import process was designed for various use-cases, like importing from multiple data sources, reoccurring import of the same data source and importing millions of objects.

We recommend our users to map a single import source to a single designated destination prefix in a lakeFS repository. When a prefix is chosen as the destination in lakeFS, the imported objects will replace any existing data in that prefix, whether it was imported or not. This enables the mirroring of the import source contents at the time of the import in the destination prefix, and allows your applications to work with the up-to-date data using lakeFS. Therefore we also recommend separating by prefix the imported data from the written data.

For example, assuming we have 2 datasets we want to import under `s3://data/source`, `cars` which are under `s3://data/source/cars/` and fruits which are under `s3://data/source/fruits/`. Assuming no other datasets are in the `source` prefix, I can import `s3://data/source/` to `lakefs://repo/main/imported/` and have the most recent `cars` & `fruits` datasets available in `lakefs://repo/main/imported/cars/` & `lakefs://repo/main/imported/fruits/`. Later I can decide to repeat the same import to retrieve the newest version of both datasets, a single data set(Importing `s3://data/source/fruits/` to `lakefs://repo/main/imported/fruits/`), or even just a single partition under a dataset (Importing `s3://data/source/fruits/type=grapes/` to `lakefs://repo/main/imported/fruits/type=grapes/`)
```

Reply:
Why wouldn't it work?
I think as long as you import to an empty branch, this all works, over and over again (adding data). 

IMHO we should not recommend our user to "map a single import source to a single designated destination", because we know that is not necessarily the use case. Instead - maybe we should teach them how to achieve what they actually want to achieve?